.gitignore

```diff
@@ -54,10 +54,13 @@ Carthage/Build
 
 # fastlane
 #
-# It is recommended to not store the screenshots in the git repo. Instead, use fastlane to re-generate the 
+# It is recommended to not store the screenshots in the git repo. Instead, use fastlane to re-generate the
 # screenshots whenever they are needed.
 # For more information about the recommended setup visit:
 # https://github.com/fastlane/fastlane/blob/master/docs/Gitignore.md
 
 fastlane/report.xml
 fastlane/screenshots
+
+# Xcode 8 playground stuff
+playgroundController.remap
```

Comment:
this lil file popped up in xcode 8 and not sure if it should be kept or not. it's contents didn't seem very interesting so gonna ignore for now